locale/en/corteza-webapp-compose/module.yaml

```diff
@@ -165,6 +165,20 @@ edit:
           label: Soft duplicate value validation
           description: Record will be saved and user will be presented with a warning when a duplicate value is detected in the selected fields in any existing record of this module
 
+    uniqueValues:
+      title: Unique values
+      allowRecordsSave: Allow record save if duplicates are found
```

Comment:
Probably a better sentence is, "Allow record saving if duplicate values are found"